src/main/java/bio/terra/stairway/Flight.java

```diff
@@ -181,7 +182,8 @@ private StepResult stepWithRetry() throws InterruptedException {
             } catch (Exception ex) {
                 // The purpose of this catch is to relieve steps of implementing their own repetitive try-catch
                 // simply to turn exceptions into StepResults.
-                logger.info("Caught exception executing flight id: " + context().getFlightId() +
+                logger.info("Caught exception: (" + ex.toString() +
```

Comment:
Is there a reason why you're calling `Logger.info(String)` instead of `Logger.info(String, Throwable)`? That would avoid the need to use `printStackTrace()` (which goes to stdout, which isn't necessarily the same as the logger output anyway) and also avoids the need to include the exception text here.

Reply:
Uh, because I didn't know about that call signature... Thanks!